### PR TITLE
[TC-85] Dashboard UI 

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[TC-0085](https://bcdevex.atlassian.net/browse/TC-0085)

Objective:

- Dashboard UI (based on first designs)
- Pagination

### CODE REVIEW

File: frontend/src/common/interface.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ClassificationName,
+  ExperienceName,
+  FunctionName,
+  MinistryAcronymName,
+  RegionName,
+  StatusName,
+  WorkLocationName,
+} from './enums';
+
+export interface Row {
```
Comment (fw-noel) [inline frontend/src/common/interface.ts:11]: Can we name DashboardRow? For searchability.

File: frontend/src/common/enums/ministry.enum.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export enum MinistryAcronymName {
+  AGRI = 'AGRI',
```
[the remark this answers is not in the document]
Comment (chelsea-EYDS) [reply frontend/src/common/enums/ministry.enum.ts:2]: I believe the plan is to use the acronyms in order to save space. It is documented... somewhere...

[the remark this answers is not in the document]
Comment (fw-noel) [reply frontend/src/common/enums/ministry.enum.ts:2]: Pop it into the decision log if you find it!

**Review by fw-noel — APPROVED** [review]

File: frontend/src/common/enums/ministry.enum.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (chelsea-EYDS) [reply frontend/src/common/enums/ministry.enum.ts:2]: Found in ticket criteria. Will add to the decision log now :)

**Pull request merged**